odxtools/envdatadesc.py

```diff
@@ -52,31 +52,21 @@ def from_et(et_element, doc_frags: List[OdxDocFragment]) \
             assert ref is not None
             env_data_refs.append(ref)
 
-        # ODX 2.0.0 says ENV-DATA-DESC could contain a list of ENV-DATAS
-        env_datas = []
-        for env_data_elem in et_element.iterfind("ENV-DATAS/ENV-DATA"):
-            env_data = EnvironmentData.from_et(env_data_elem, doc_frags)
-            env_datas.append(env_data)
```

Comment:
Not parsing this anymore?

Reply:
oops. ok, odx 2.0. I didn't find a ENV-DATAS sub-element in the XSD. I'll revert.